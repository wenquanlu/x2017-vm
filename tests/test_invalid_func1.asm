FUNC LABEL 0
    MOV STK A VAL 10
    REF REG 1 STK A
    CAL VAL 1
    RET
FUNC LABEL 2
    MOV STK A REG 1
    RET
