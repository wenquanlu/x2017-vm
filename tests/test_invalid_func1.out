Function being called not defined
